ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.13.yaml

```diff
@@ -101,7 +101,7 @@ tests:
   steps:
     cluster_profile: aws-qe
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: CloudCredential CloudControllerManager Ingress
+      ADDITIONAL_ENABLED_CAPABILITIES: Console Insights openshift-samples
```

Comment:
Per the source version setting (`aws-upi-basecap-none-arm-f28` in openshift-openshift-tests-private-release-4.13__multi-nightly.yaml), the job does NOT need any setting for ADDITIONAL_ENABLED_CAPABILITIES, right?

Reply:
I don't remember why this setting was added, just checked https://github.com/openshift/release/pull/53849, there is no comments on it. It seems a mistake.

Reply:
Removed it